tools/mo/openvino/tools/mo/back/MaxPool.py

```diff
@@ -40,7 +40,7 @@ def normalize_outputs(node: Node):
                                          'keep_output_port': node.has_and_set('remove_values_output')}).create_node()
             node.out_port(0).get_connection().set_destination(output.in_port(0))
 
-        if node.out_port(1).disconnected():
+        if node.has_port('out', 1) and node.out_port(1).disconnected():
```

Comment:
why did it happen that port 1 is absent if output ports number is set to 2 in Pooling class?

Reply:
According to [MaxPool_1](https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/pooling/MaxPool_1.md) specification we must have only 1 output and 2 for [MaxPool_8](https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/pooling/MaxPool_8.md)

Reply:
Add comment with explaining as we discussed